components/eamxx/src/share/io/CMakeLists.txt

```diff
@@ -15,8 +15,8 @@ set_target_properties(scream_io PROPERTIES
 )
 target_include_directories(scream_io PUBLIC ${CMAKE_CURRENT_BINARY_DIR}/modules)
 
-if (DEFINED ENV{ADIOS2_DIR})
-  target_include_directories(scream_io PRIVATE $ENV{ADIOS2_DIR}/include)
+if (DEFINED ENV{ADIOS2_ROOT})
```

Comment:
Should we instead add this to the [Scorpio.cmake](https://github.com/E3SM-Project/E3SM/blob/master/components/eamxx/cmake/tpls/Scorpio.cmake) target?

Reply:
Probably, but we can refactor that on the scream side.